### PR TITLE
Add test case performing repeated blitFramebuffer calls without clears.

### PR DESCRIPTION
**Pull request by kenrussell**
Test case for crbug.com/768969 .

### CODE REVIEW

**Comment by kenrussell** [comment]
Note: this test indicates there's no bug in Chromium.

**Comment by kenrussell** [comment]
@zhenyao @jdashg in particular please take a look.

I suspect this is already covered by other conformance tests but didn't confirm that.

**Comment by zhenyao** [comment]
LGTM

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html [hunk 1]
```diff
@@ -52,15 +52,56 @@
...
 }
 
+function blit_framebuffer_repeated() {
+    debug("");
+    debug("This test verifies repeated calls to blitFramebuffer.");
+
+    // Create offscreen fbo and its color attachment.
+    var tex_2d = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, tex_2d);
+    gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA8, width, height, 0, gl.RGBA, gl.UNSIGNED_BYTE, null);
```
Comment (jdashg) [inline sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Slight preference for texStorage in webgl2-only code.

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html [hunk 2]
```diff
@@ -52,15 +52,56 @@
...
+    wtu.setFloatDrawColor(gl, [ 1.0, 0.0, 0.0, 1.0 ]);
+    wtu.drawUnitQuad(gl);
+    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, fb);
+    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, null);
+    gl.blitFramebuffer(0, 0, width, height, 0, 0, width, height, gl.COLOR_BUFFER_BIT, gl.NEAREST);
+    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, null);
+    wtu.checkCanvas(gl, [ 255, 0, 0, 255 ], "should be red at first");
+
+    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, fb);
+    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, null);
```
Comment (jdashg) [inline sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Skip setting READ_FB if we aren't going to read from it.
Generally, choose to either default to using FRAMEBUFFER except for calls to BlitFramebuffer, or only set the READ_/DRAW_FRAMEBUFFER that you need for the next command.

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html (hunk 1 above)
Comment (kenrussell) [reply sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Fixing in #2569.

File: sdk/tests/conformance2/rendering/blitframebuffer-test.html (hunk 2 above)
Comment (kenrussell) [reply sdk/tests/conformance2/rendering/blitframebuffer-test.html]: Fixed throughout this test in #2569.

**Pull request merged**